components/automate-ui/src/styles.scss

```diff
@@ -328,3 +328,21 @@ input {
     display: none;
   }
 }
+
+.waived-icon {
```

Comment:
@jonong1972 what would you think of adding a 2px border-radius to match with the other rectangular material icons?

Reply:
@susanev sure! sounds good to me

Reply:
@susanev added border radius 😊